utils/kind-cluster.yaml

```diff
@@ -4,9 +4,14 @@ apiVersion: kind.x-k8s.io/v1alpha4
 nodes:
 - role: control-plane
   image: kindest/node:v1.27.3
-  # port forward 80 on the host to 80 on this node
   extraPortMappings:
     - containerPort: 30950
       hostPort: 9080
     - containerPort: 30951
       hostPort: 9443
```

Comment:
If we decide to use metalb in the default setup we can probably drop these ports and use 9080 -> 80, 9443 -> 443